packages/styles/button.css

```
@@ -10,7 +10,7 @@
   --button-background-color-secondary-active: var(--gray-30);
 
   --button-background-color-error: var(--error);
-  --button-background-color-error-disabled: #db6379;
+  --button-background-color-error-disabled: var(----accent-error-disabled);
```

Is this supposed to be this?

```suggestion
  --button-background-color-error-disabled: var(--accent-error-disabled);
```

I do appreciate syncing missing vars, but I think it might be better to split this into 2 PRs to isolate unrelated changes. A `feat` PR to implement any missing tokens/css vars and a `docs` PR to actually document the changes.